### ISSUE TITLE
Synchronize Bintray repo with Maven Central

### ISSUE THREAD

**Issue by marcingrzejszczak** [4finance/micro-infra-spring#103]

**Comment by nurkiewicz** [comment]
Aforementioned ticket is closed, can we look at Maven Central synchronization again? With automatic release management it's especially compelling.

**Comment by szpak** [comment]
Yes, it was implemented in 1.0 and I see no more technical obstacles.

**Comment by nurkiewicz** [comment]
> I see no more technical obstacles

Well, there is one: `com.ofg` package, that we don't own. We can technically rename artifacts to `groupId` `com.4financeit`.

**Comment by marcingrzejszczak** [comment]
@achudzik will investigate the domain availability and stuff so I'm assigning him to the task

**Comment by marcingrzejszczak** [comment]
the chosen package is

*io.fourfinance*
